src/rpc/rawtransaction.cpp

```diff
@@ -39,6 +39,7 @@
 
 #include <univalue.h>
 
+constexpr static CAmount DEFAULT_MAX_RAW_TX_FEE{COIN / 10};
```

Comment:
There is already `DEFAULT_TRANSACTION_MAXFEE` in validation.h, isn't a new constant confusing?